perf/index.js

```diff
@@ -0,0 +1,63 @@
+const levenshteinEditDistance = require('levenshtein-edit-distance');
+const fastLevenshtein = require('fast-levenshtein');
+const jsLevenshtein = require('js-levenshtein');
+const talisman = require('talisman/metrics/distance/levenshtein');
+const leven = require('leven');
+const jsLevenshteinEsm = require('esm')(module)('../');
+const {paragraphs, sentences, words} = require('esm')(module)('./data');
+
+if (!Object.entries) {
+  Object.entries = function( obj ) {
```

Comment:
did something _try_ to format this? what happened with the spacing here?

Reply:
pasted from mdn